app/browser/reducers/historyReducer.js

```diff
@@ -126,6 +127,24 @@ const historyReducer = (state, action, immutableAction) => {
         break
       }
 
+    case appConstants.APP_REMOVE_HISTORY_DOMAIN: {
+      const domain = action.get('domain')
+
+      if (!domain) {
+        break
+      }
+
+      historyState.getSites(state).forEach(historySite => {
+        if (urlParse(historySite.get('location')).hostname === domain) {
```

Comment:
should probably check that `urlParse(historySite.get('location'))` is non-falsey

Reply:
it seems `urlParse` always returns an object

https://github.com/brave/browser-laptop/blob/master/app/common/urlParse.js#L19

So it should always be non-falsey

Would you still prefer a check in there? If so should we display any errors to the user? Or just ignore the fail and do nothing?

Reply:
Any thoughts on this @diracdeltas

Reply:
nvm original comment